client/core/core.go

```diff
@@ -4058,16 +4060,26 @@ func (c *Core) Withdraw(pw []byte, assetID uint32, value uint64, address string)
 	if err != nil {
 		return nil, err
 	}
+	var coin asset.Coin
 	feeSuggestion := c.feeSuggestionAny(assetID)
-	coin, err := wallet.Withdraw(address, value, feeSuggestion)
+	if !subtract {
+		coin, err = wallet.Wallet.Send(address, value, feeSuggestion)
+	} else {
+		if withdrawer, isWithdrawer := wallet.Wallet.(asset.Withdrawer); isWithdrawer {
+			coin, err = withdrawer.Withdraw(address, value, feeSuggestion)
+		} else {
+			return nil, fmt.Errorf("wallet does not support subtracting network fee from withdraw amount")
```

Comment:
this return prevents the notification below from being sent, is this intentional ? I don't know who exactly expects what notifications, 
but I think it makes sense to send error notification here as well, since other errors are notified on as well

```go
subject, details := c.formatDetails(TopicSendError, unbip(assetID), err)
c.notify(newSendNote(TopicSendError, subject, details, db.ErrorLevel))
```